### PR TITLE
fix tempdir examples

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: cromwell.examples.conf [hunk 1]
```diff
@@ -277,13 +277,13 @@ backend {
         # `temporary-directory` creates the temporary directory for commands.
         #
         # If this value is not set explicitly, the default value creates a unique temporary directory, equivalent to:
-        # temporary-directory = "mktemp -d \"$PWD\"/tmp.XXXXXX"
+        # temporary-directory = "$(mktemp -d \"$PWD\"/tmp.XXXXXX)"
```
Comment (cjllanwarne) [inline cromwell.examples.conf]: I think this example should be below the "what does this value mean" explanation below

File: cromwell.examples.conf [hunk 2]
```diff
@@ -277,13 +277,13 @@ backend {
...
         #
         # The expression is run from the execution directory for the script. The expression must create the directory
         # if it does not exist, and then return the full path to the directory.
         #
         # To create and return a non-random temporary directory, use something like:
-        # temporary-directory = "mkdir -p /tmp/mydir && echo /tmp/mydir"
+        # temporary-directory = "$(mkdir -p /tmp/mydir && echo /tmp/mydir)"
```
Comment (cjllanwarne) [inline cromwell.examples.conf]: ToL is it not possible for Cromwell to just auto-wrap these values as it reads them in? Right now this just feels like a weird thing you have to to remember to do (differently from before) in the config file

File: cromwell.examples.conf (hunk 1 above)
Comment (cjllanwarne) [inline cromwell.examples.conf]: Does this count as a breaking change?

Comment (mcovarr) [reply cromwell.examples.conf]: This counts as unbreaking an [unintentionally breaking change](https://github.com/broadinstitute/cromwell/issues/3784) 😛

**Pull request merged**